jobs/clamav/templates/bin/pre-start.erb

```diff
@@ -2,6 +2,21 @@
 
 export MINUTE=$(expr $RANDOM % 60)
 export HOUR=$(shuf -i <%= p('clamav.nightly.scan.start') %>-<%= p('clamav.nightly.scan.stop') %> -n 1)
+export CLAMAV_HOME=/var/vcap/packages/clamav
+export DATADB_DIR=/var/vcap/data/clamav/database
+
+# make database DIR if missing
+if [ ! -d "${DATADB_DIR}" ]; then
+	mkdir -p ${DATADB_DIR}
+fi
```

Comment:
the test for datadb_dir is unneccesary, right? we could just `mkdir -p`

Reply:
@bengerman13 I just followed what existed in the script already.  I also think it's cleaner to check and less red herring with a `mkdir` error in log output.

Reply:
`mkdir -p` won't complain about existing directories :)

Reply:
ahh good to know, I can change that then.